packages/orbit-components/src/common/placements.ts

```diff
@@ -0,0 +1,39 @@
+import type { Placement as FloatingPlacement, Alignment } from "@floating-ui/react";
+
+export type AutoPlacement = "auto" | "auto-start" | "auto-end";
+export type Placement = FloatingPlacement;
+
+export enum AUTO_PLACEMENTS {
+  AUTO = "auto",
+  AUTO_START = "auto-start",
+  AUTO_END = "auto-end",
+}
+
+export enum PLACEMENTS {
+  TOP = "top",
+  TOP_START = "top-start",
+  TOP_END = "top-end",
+  BOTTOM = "bottom",
+  BOTTOM_START = "bottom-start",
+  BOTTOM_END = "bottom-end",
+  RIGHT = "right",
+  RIGHT_START = "right-start",
+  RIGHT_END = "right-end",
+  LEFT = "left",
+  LEFT_START = "left-start",
+  LEFT_END = "left-end",
+}
```

Comment:
```suggestion
}
export enum PLACEMENT = {
   ...FIXED_PLACEMENT,
   ...AUTO_PLACEMENT,
}
```

Reply:
I don't think we can spread enums 🤔

Reply:
You're right, sorry.

We could imagine instead something like: https://www.typescriptlang.org/play/?ssl=21&ssc=27&pln=1&pc=1#code/KYOwrgtgBAouEEYoG8BQUoEEoF4oHJN8AadKAIVwPJNQF9VVRJZ4AmFMgYSvy9owARXoNoNUAYwD2IAM4AXKAFlgAJwDmwACZwWeNBgB0x3YlJGT7UnSgBDWVGlz5AbkbyAngAdgytZp14ABVvXzxPHykAMz8NbVM3RgB6JKgAVVlbTUkZBSgAN1sAGzBgAC5YgNMQnwBtAGtgD2jK+ODQgF0qFTjAyENMNydZKSLgQyKpdQAKQpLgAEoXKBSCIiA

```suggestion
const _placement = {
   ...FIXED_PLACEMENT,
   ...AUTO_PLACEMENT,
} as const

export type PLACEMENT = typeof _placement
```

Reply:
I also couldn't achieve a valid definition

https://www.typescriptlang.org/play/?#code/KYOwrgtgBAouEEYoG8BQUoEEoF4oHJN8AadKAIVwPJNQF9VVRJZ4AmFMgYSvy9owARXoNoNUAYwD2IAM4AXKAFlgAJwDmwACZwWeNBgB0x3YlJGT7UnSgBDWVGlz5AbkbyAngAdgytZp14ABVvXzxPHykAMz8NbVM3RgB6JKgAVVlbTUkZBSgAN1sAGzBgAC5YgNMQnwBtAGtgD2jK+ODQgF0qFTjAyENMNydZKSLgQyKpdQAKQpLgAEoXKBSCIkZgAA8vKVVFZmhMNKCAeQB9AAUAGUwuGCUYADkggGVODCPTqgAiWzB5KTfcxYY7nF5BTAAJSCPz+AIAtApbHsgWRPucnsI8L9-lJ4aAtKjxFsdnsoAcoAAxACSAA0YIJLjc7g9nm8DFBThcfgCvKiMFyzuCoTDsbzEfJkfJ+ZyThczpieVIvPiQITgeQTkFTkofgAjKTyAEQGWa7UnJRCiHQ-WG40SqWmrU6hWPLFQb4Go1SCCq9VkSHUgDiAAlRR7VABLdQAC2lwMDoaCVpFPyjsfkDpRCeDYdd7u+6bjfplVxglPD3zGUXjZDLFZTNux1czSOzdfLycVzeANZL1kYw0UZy8RVsEmAEFAin0ZCgxkMNPpjOut3uT1ewPnxnRTLXrM39DsDiHG22u0UEV8q5ZG6oV5aI7HE6nIHkg9yik2FRv6+eVCXBk91vNlDC5D9nCgDwf2ZP9K15b4ILyAB3GD9zvPAn3HSdpzAuUkNGcZJhmTYFhyORCImKZpg8Mjhko4jpmQhYgA

Reply:
Very well. Ignore my suggestions then 😅 thanks for trying!